src/components/Icon/Icon.js

```diff
@@ -1,7 +1,11 @@
+// @flow
+
 import React from 'react'
 import Icons from 'react-native-vector-icons/FontAwesome5'
 
-const Icon = ({ name, size = 40, color = '#ffffff', style, }) => {
+type Props = { name: any, size: number, color: string, style: mixed }
```

Comment:
Better `IconProps`

Reply:
`name: any`
Why not `string`?

Reply:
Because Icon for the property name use own literal types

Reply:
and flow throw an error when i use the 'string' type

Reply:
@Dekim12 Maybe you could import them?